src/plotting.jl

```diff
@@ -376,8 +376,8 @@ end
 Create a Nyquist plot of the `LTISystem`(s). A frequency vector `w` can be
 optionally provided.
 
-`Ms_circles`, vector of Ms values for drawing circles of constant sensitivity
-`unit_circle`, if the unit circle should be displayed
+- `unit_circle`: if the unit circle should be displayed
+- `Ms_circles`: denotes maximum allowed values of the sensitivity function, a circle around -1 will be drawn with `1/Ms` radius. `Ms_circles` can be supplied as a number or a vector of numbers. A design staying outside such a circle has a phase margin of at least `2asin(1/(2Ms))` rad and a gain margin of at least `Ms/(Ms-1)`.
```

Comment:
```suggestion
- `Ms_circles`: draw circles corresponding to given levels of sensitivity (circles around -1 with  radii `1/Ms`). `Ms_circles` can be supplied as a number or a vector of numbers. A design staying outside such a circle has a phase margin of at least `2asin(1/(2Ms))` rad and a gain margin of at least `Ms/(Ms-1)`.
```
Feel free to disregard.